testsuite/features/secondary/srv_restart.feature

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Restart the spacewalk-services via UI
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Admin > Manager Configuration > Restart"
+    And I check "restart"
+    And I click on "Restart"
+    And I wait for "300" seconds
```

Comment:
I personally don't like to use implicit waits, better use the step above with a timeout of 300s.
So if it finish earlier, we earn some seconds.

Reply:
you mean "below" ?

Reply:
hehe yes bellow :)
https://www.youtube.com/watch?v=YRa7oYLehA0

Reply:
:-)